projects/angular/src/data/tree-view/_tree-view.clarity.scss

```diff
@@ -31,7 +31,7 @@
 
         cds-icon,
         clr-icon {
-          fill: var(--clr-tree-node-caret-disabled-color);
+          --color: var(--clr-tree-node-caret-disabled-color);
```

Comment:
This won't work in places we allow users project a `clr-icon`. We can only use `--color` when we own the element and know it will be a `cds-icon`.